pkg/apis/core/validation/shoot.go

```diff
@@ -51,6 +52,8 @@ import (
 )
 
 var (
+	// maxOperationsSize is the maximum allowed length of the operations annotation in bytes.
+	maxOperationsSize   = 1 * 8 * 1024
```

Comment:
Nit: 
```suggestion
	maxOperationsSize   = 8 * 1024
```

Also could you post some findings about resource consumption now that we have this limit in place?